root_VS2015/programs/VB/7_Build_Framework_WS.bat

```diff
@@ -16,16 +16,9 @@ set CURRENT_DIR="%~dp0"
 call %CURRENT_DIR%z_Common.bat
 
 rem --------------------------------------------------
-rem Copy the Bin folder assembly.
-rem --------------------------------------------------
-
-md "Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\Bin"
-xcopy /E /Y "Samples\WS_sample\Build" "Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\Bin\"
-pause
-
-rem --------------------------------------------------
 rem Batch build of ServiceInterface(ASPNETWebService).
 rem --------------------------------------------------
+..\nuget.exe restore "Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService.sln"
 %BUILDFILEPATH% %COMMANDLINE% "Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService.sln"
```

Comment:
@daisukenishino 

The build of this solution fails.
Please confirm.

```txt
--------------------------------------------------
The choice of build configuration (Debug / Release).
--------------------------------------------------
--------------------------------------------------
Creating a build command.
--------------------------------------------------
--------------------------------------------------
Set the proxy settings of Nuget.
--------------------------------------------------
MSBuild auto-detection: using msbuild version '14.0' from 'C:\Program Files (x86)\MSBuild\14.0\bin'.
Microsoft (R) Build Engine バージョン 4.7.2046.0
[Microsoft .NET Framework、バージョン 4.0.30319.42000]
Copyright (C) Microsoft Corporation. All rights reserved.

このソリューション内のプロジェクトを 1 度に 1 つずつビルドします。並行ビルドを有効にするには、"/m" スイッチを追加してください。
2017/09/26 14:16:37 にビルドを開始しました。
ノード 1 上のプロジェクト "...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService.sln" (既定のターゲット)。
ValidateSolutionConfiguration:
  ソリューション構成 "Debug|Any CPU" をビルドしています。
"C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\Web\Microsoft.Web.Publishing.targets (852,131)" にある BeforeTargets 属性に一覧表示されているターゲット "MvcBuildViews" はプロジェクト内に存在しないため、無視されます。
プロジェクト "...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService.sln" (1) は、ノード 1 上に "...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService\ASPNETWebService.csproj" (2) をビルドしています (既定のターゲット)。
プロジェクト ファイルに ToolsVersion="12.0" が含まれています。このツールセットが不明であるか、存在しない可能性があります。その場合は、適切なバージョンの MSBuild をインストールして解決するか、ビルドがポリシー上の理由により特定の ToolsVersion を使用するよう強制されている可能性があります。プロジェクトを ToolsVersion="4.0" として扱います。詳細については、http://go.microsoft.com/fwlink/?LinkId=291333 を参照してください。
...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService\ASPNETWebService.csproj(224,5): error : このプロジェクトは、このコンピューター上にない NuGet パッケージを参照しています。それらのパッケージをダウンロードするには、[NuGet パッケージの復元] を使用します。詳細については、http://go.microsoft.com/fwlink/?LinkID=322105 を参照してください。見つからないファイルは ..\packages\Microsoft.Net.Compilers.1.0.0\build\Microsoft.Net.Compilers.props です。
_CleanRecordFileWrites:
  ディレクトリ "obj\Debug\" を作成しています。
プロジェクト "...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService\ASPNETWebService.csproj" (既定のターゲット) のビルドが終了しました -- 失敗。
プロジェクト "...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService.sln" (既定のターゲット) のビルドが終了しました -- 失敗。

ビルドに失敗しました。

"...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService.sln" (既定のターゲット) (1) ->
"...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService\ASPNETWebService.csproj" (既定のターゲット) (2) ->
(EnsureNuGetPackageBuildImports ターゲット) -> 
  ...\root_VS2015\programs\C#\Frameworks\Infrastructure\ServiceInterface\ASPNETWebService\ASPNETWebService\ASPNETWebService.csproj(224,5): error : このプロジェクトは、このコンピューター上にない NuGet パッケージを参照しています。それらのパッケージをダウンロードするには、[NuGet パッケージの復元] を使用します。詳細については、http://go.microsoft.com/fwlink/?LinkID=322105 を参照してください。見つからないファイルは ..\packages\Microsoft.Net.Compilers.1.0.0\build\Microsoft.Net.Compilers.props です。

    0 個の警告
    1 エラー

経過時間 00:00:00.16
続行するには何かキーを押してください . . . 
```

Reply:
@daisukenishino 

It seems that this build error is caused by the length of the folder path.
I create the [new issue](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/71).